utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/swagger/SwaggerTyped.scala

```diff
@@ -87,6 +86,8 @@ sealed trait SwaggerUnknownFallback extends SwaggerTyped
 
 case object SwaggerRecursiveSchema extends SwaggerUnknownFallback
 
+case object SwaggerAny extends SwaggerUnknownFallback
```

Comment:
One slight detail. IMO after this change `SwaggerUnknownFallback` has no sence. It was fallback fall situation when recursive schema was detected. Maybe we should have `SwaggerAny` trait and two objects showing in which case it was used: `SwaggerRecursiveSchema` and `SwaggerAny` (the same name, but object). WDYT?